lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Human < Player
+
+  def move(board)
+    puts "What move would you like to make (1-9)?"
+    move = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "What move would you like to make (1-9)?"
  gets.chomp
end